packages/react-scripts/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # `backpack-react-scripts` Change Log
 
+## (Pending)
+
```

Comment:
> We will probably need to mention here the new config for SSR in order to turn it on based on the `isSSR` function.
> 
> We will also need to remember to mark this release as breaking as it will require SSR users to add a new config item otherwise it will cause this to break for them.

The SSR projects is few, `false` default value is intentionally set here, this will make a deeper impression on engineers for they should know what they are doing is to make projects run with SSR mode.

Reply:
> We will probably need to mention here the new config for SSR in order to turn it on based on the `isSSR` function.
> 
> We will also need to remember to mark this release as breaking as it will require SSR users to add a new config item otherwise it will cause this to break for them.

Yep, will add new guidance here for SSR projects before merging to `fork`, not sure if the version of BRS with sassFunctions implemented by itself will be faster than this PR. Just a memo here.